Makefile

```diff
@@ -14,12 +14,16 @@ xcodeproj:
 	swift package generate-xcodeproj --xcconfig-overrides Development.xcconfig
 	xed .
 
-bootstrap: common-crypto-mm xcodeproj
+bootstrap: common-crypto-mm xcodeproj xcodeproj-mm
 
 common-crypto-mm:
 	-@sudo mkdir -p "$(COMMON_CRYPTO_PATH)"
 	-@echo "$$COMMON_CRYPTO_MODULE_MAP" | sudo tee "$(COMMON_CRYPTO_MODULE_MAP_PATH)" > /dev/null
 
+xcodeproj-mm:
```

Comment:
This is a fun one. It installs any module maps that are generated with the Xcode project.

Reply:
oh nice, can that replace our other modulemap copying?